### DISCUSSION TITLE
v1.7.0

### DISCUSSION THREAD

**Discussion by JunkFood02** [JunkFood02/Seal#461] [Announcements]
## What's Changed
Feature Update:
- Generate, store and use cookies for multiple websites using built-in WebView
- Select from the formats of videos to download
- Run custom commands in parallel in the background
- Register to open URLs from other apps
- Aria2 now can be used with custom commands

Bug Fix:
- "File name too long" error when downloading videos with long titles

UI Improvements:
- Optimize text display in multiple languages
- Add help messages in setting pages
- Show total file size when deleting multiple files

Behavior Changes:
- Embedded artwork is converted to jpeg to lower the size of file
- Devices running Android 11+ (API 30+) can grant Seal `MANAGE_EXTERNAL_STORAGE` permission to access other directories than `Download/`
- Auto-update is enabled by default, updating yt-dlp along with the app

Translate Update:
- Add Norwegian Nynorsk translations by @tryvseu
- Add Azerbaijani translations
- Add Punjabi translations by @ShareASmile

Special Thanks:
- All the beta testers!

**Full Changelog**: https://github.com/JunkFood02/Seal/compare/v1.6.1...v1.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/JunkFood02/Seal/releases/tag/v1.7.0'>v1.7.0</a>.</em>

**Comment by bikram-agarwal** [comment]
On Pixel 7 Pro, running Android 13 QPR2 Beta 1, when updating from previous version to this, I'm getting an error saying "_App not installed as app isn't compatible with your phone._" Is this version 32 bit only, by any chance?

**Reply by JunkFood02** [reply]
Maybe there's sth wrong with the auto-update making it downloading the wrong apk, you can install the armv8a one manually

**Reply by bikram-agarwal** [reply]
Yep, that worked. Directly downloaded the apk from release page and that updated the previous installation fine.

**Reply by JunkFood02** [reply]
I've pushed a hotfix for this, but you may have the same issue when auto-updating to the next stable version. btw Pixel 7 is indeed a good device

**Comment by rafit612** [comment]
Show problem in android 13 bro,,,can you do something

**Comment by gyan-io** [comment]
It's not compatible with my phone

**Reply by gyan-io** [reply]
Samsung A02s

**Reply by JunkFood02** [reply]
Try both arm64-v8a and armeabi-v7a

**Comment by oop7** [comment]
why not download the audio in high quality ?

**Reply by Marco-9456** [reply]
> why not download the audio in high quality ?

Because it doesn't have access to it,
But if you have a Youtube Premium Subscription then just generate Cookies for it and `high quality Audio` will appear.

**Reply by oop7** [reply]
Thank you so much
